README.md

```diff
@@ -1,7 +1,228 @@
+* Allows user to persist the settings for a plug-in response. 
+* Allows plugins to customize their own UI.
+
+## Supported plugins
+
+This preview version currently includes built-in support for the following Cordova plugins:
+
+* [cordova-plugin-camera](https://github.com/apache/cordova-plugin-camera)
+* [cordova-plugin-console](https://github.com/apache/cordova-plugin-console)
+* [cordova-plugin-contacts](https://github.com/apache/cordova-plugin-contacts)
+* [cordova-plugin-device](https://github.com/apache/cordova-plugin-device)
+* [cordova-plugin-device-motion](https://github.com/apache/cordova-plugin-device-motion)
+* [cordova-plugin-dialogs](https://github.com/apache/cordova-plugin-dialogs)
+* [cordova-plugin-file](https://github.com/apache/cordova-plugin-file)
+* [cordova-plugin-geolocation](https://github.com/apache/cordova-plugin-geolocation)
+* [cordova-plugin-globalization](https://github.com/apache/cordova-plugin-globalization)
+* [cordova-plugin-media](https://github.com/apache/cordova-plugin-media)
+* [cordova-plugin-vibration](https://github.com/apache/cordova-plugin-vibration)
+
+## Adding simulation support to plugins
+
+It also allows for plugins to define their own UI. To add simulation support to a plugin, follow these steps:
+
+1. Clone this repository (`git clone https://github.com/microsoft/cordova-simulate.git`), as it contains useful example code (see `src/plugins`).
+2. Add your plugin UI code to your plugin in `src/simulation`. Follow the file naming conventions seen in the built-in plugins.
+
+### Detailed steps
+
+In your plugin project, add a `simulation` folder under `src`, then add any of the following files:
+
+```
+sim-host-panels.html
+sim-host-dialogs.html
+sim-host.js
+sim-host-handlers.js
+app-host.js
+app-host-handlers.js
+app-host-clobbers.js
+```
+ 
+#### Simulation Host Files
+
+*sim-host-panels.html*
+
+This defines panels that will appear in the simulation host UI. At the top level, it should contain one or more
+`cordova-panel` elements. The `cordova-panel` element should have an `id` which is unique to the plugin (so the plugin
+name is one possibility, or the shortened version for common plugins (like just `camera` instead of
+`cordova-plugin-camera`). It should also have a `caption` attribute which defines the caption of the panel.
+ 
+The contents of the `cordova-panel` element can be regular HTML, or the various custom elements which are supported
+(see the existing plugin files for more details).
+ 
+This file shouldn't contain any JavaScript (including inline event handlers), nor should it link any JavaScript files.
+Any JavaScript required can be provided in the standard JavaScript files described below, or in additional JavaScript
+files that can be included using `require()`.
+ 
+*sim-host-dialogs.html*
+
+This defines any dialogs that will be used (dialogs are simple modal popups � such as used for the Camera plugin). At
```

Comment:
Something got screwed with the readme in its most recent update. Could you change each � back into a dash?